tests/flux-automation.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -x
+
+_github_sha=$1
+
+whitelist_dirs=(
+    admin
+    kube-system
+    kube-public
+    knode-system
+    neuvector)
+
+[ -z "$_github_sha" ] && echo "Error: github commit sha missing." && exit 1
+
+_errors=""
+
+for f in $(git diff-tree --no-commit-id --name-only -r $_github_sha)
+do
+  # check if whitelisted
+  for wd in "${whitelist_dirs[@]}"
+  do 
+    echo "$f" | grep -E -q "k8s/(aat|prod)/(common|cluster-00|cluster-01)/${wd}"
+    [ $? -eq 0 ] && continue 2
+  done
+  # check if automated
+  fgrep -E -q '(flux\.weave\.works|fluxcd\.io)/automated: *"true"' "$f"
```

Comment:
would be good to cleanup the deprecated annotations 👅

Reply:
True ... but not in this story. Also this script needs to check for it anyway (in case it's still in use).